desks.json

```diff
@@ -0,0 +1,122 @@
+{
+  "Biurko nr 1": {
```

Comment:
zamiast pisac "Biurko numer 1" po porstu pisał bym 1: bo zrobicie literowke i sie porobią blędy. Biurka nr 1 to tylko nazewnictow sam numer jest indetyfikatorem.